dist/README.UPDATERS

```diff
@@ -25,36 +30,24 @@ Note: * Update from OBS 2.5 should also work, but is untested.
 
         zypper dup --from OBS_Server_2.10
 
-# Needed in case of a ruby update
-#
-# 5) Change to ruby2.5
-#
-#       edit /etc/apache2/conf.d/mod_passenger.conf:
-#
-#       PassengerRuby "/usr/bin/ruby.ruby2.5"
+    NOTE: You may need to switch also your base distribution to a newer version. Adapt all repositories for that
+          and do an
+
+        zypper dup
 
-  6) Migrate database
+      instead of the above command
+
+  5) Migrate database
 
         cd /srv/www/obs/api/
         RAILS_ENV="production" bin/rails db:migrate:with_data
 
-  7) Make sure that log and tmp are owned by wwwrun
+  6) Make sure that log and tmp are owned by wwwrun
 
         chown -R wwwrun.www /srv/www/obs/api/log
         chown -R wwwrun.www /srv/www/obs/api/tmp
 
-  8) Restart following services in this order
-
-        systemctl restart apache2
-        systemctl restart obs-api-support.target
-        systemctl restart memcached
-
-  9) Enable and start the new services
-
-        systemctl enable obsservicedispatch
-        systemctl start obsservicedispatch
-
-     obsservicedispatch must run on the host where the src server is running.
+  7) Reboot your system to restart all OBS components
```

Comment:
Fine with require a reboot. If we don't support upgrading without rebooting, we should also remove [this lines](https://github.com/openSUSE/open-build-service/blob/60f9ce6aa0ec62537c2dc9712cbe236551920df3/dist/obs-server.spec#L609-L613) from `obs-server.spec` file.